test/SlowTests/Issues/RavenDB-11767.cs

```diff
@@ -62,11 +62,11 @@ public void ShouldSimplifyTransparentIdentifierParameters(Options options)
                                                 }
                                     };
 
-                    Assert.Equal("from 'Orders' as o load o.CategoryListId as categoryList " +
+                     Assert.Equal("from 'Orders' as o load o.CategoryListId as categoryList " +
                                  "select { Id : id(o), Items : o.OrderItems" +
                                     ".map(i=>({i:i,cat:categoryList.Categories}))" +
                                     ".map(__rvn0=>({__rvn0:__rvn0,id:__rvn0.i.CategoryId}))" +
-                                    ".map(__rvn1=>({__rvn1:__rvn1,first:__rvn1.__rvn0.cat.find(x=>id(x)===__rvn1.id)}))" +
+                                    ".map(__rvn1=>({__rvn1:__rvn1,first:__rvn1.__rvn0.cat.find(x=>x.Id===__rvn1.id)}))" +
```

Comment:
So this is a breaking change, right?

Reply:
If that's the case then we need to mark it as breaking change on YT

Reply:
I think so.